Userland/Libraries/LibBitTorrent/Bencode/BTypes.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023, Pierre Delagrave <pierre.delagrave@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/DeprecatedString.h>
+#include <AK/HashMap.h>
+#include <AK/Variant.h>
+
+namespace BitTorrent {
+
+struct List;
```

Comment:
`using List = Vector<BEncodingType>`?

Reply:
It won't work because of the recursive nature of these data structures.
Maybe I missed something though, but after a while that's the simplest working definitions I could come up with.

Reply:
Oops, I totally missed that it was using a `Vector` of `BEncodingType`, sorry about that!
That might be the simplest thing to do then